File: 2598/2598.cs
public class Solution {
    public int SmallestMissingNonnegativeInte(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(SmallestMissingNonnegativeInte(root.left), SmallestMissingNonnegativeInte(root.right));
    }
}

File: 2598/2598.go
func smallestMissingNonnegativeInte(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := smallestMissingNonnegativeInte(root.Left)
	r := smallestMissingNonnegativeInte(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}